assets/javascripts/discourse/templates/components/chat-composer-dropdown.hbs

```diff
@@ -0,0 +1,26 @@
+{{#if buttons.length}}
+  {{#d-popover
+    class="chat-composer-dropdown"
+    options=(hash arrow=null)
+    as |state|
+  }}
+    {{flat-button
+      disabled=isDisabled
+      class="chat-composer-dropdown__trigger-btn d-popover-trigger"
+      title="chat.composer.toggle_toolbar"
+      icon=(if state.isExpanded "minus" "plus")
+    }}
+    <ul class="chat-composer-dropdown__list">
+      {{#each buttons as |button|}}
+        <li class="chat-composer-dropdown__list__item {{button.id}}">
```

Comment:
> * I'm not a huge fan of BEM naming, and we probably should decide internally whether this is our preferred class naming convention before introducing this in code

Changing it to something else will be easy and I want real life experiment, chat is still under beta I can change classes in the next weeks. Worst case it will be a few find/replace

> * shouldn't the button id be an id? feels odd to convert it to a class here

Button id doesn't meant DOM ID, I have no guarantees this will be unique on the page.